_i18n/de/resources/moneropedia/fluffyblocks.md

```diff
@@ -1,18 +1,14 @@
 ---
-terms: ["fluffy blocks", "fluffyblocks"]
-summary: "a way of saving bandwidth when sending new blocks around the network"
+terms: ["fluffy blocks", "fluffyblocks", "Fluffy-Blocks"]
+summary: "Eine Möglichkeit, beim Senden neuer Blöcke Datenübertragungsraten zu verringern"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A @block is made up of a header and @transactions. Fluffy Blocks only contain
-a header, a list of transaction indices, and any transactions that the node
-recieving the block may be missing. This saves bandwidth because nodes might
-already know about most or all of the transactions in the block and they don't
-need to be sent them again.
+Ein @Block besteht aus einem Header und @Transaktionen. Fluffy-Blocks enthalten lediglich einen Header, eine Liste von Transaktionsindexen und jegliche Transaktionen, welche der den Block empfangende Node eventuell verfehlt hat. Dies spart Bandbreite, weil Nodes möglicherweise bereits über die meisten Transaktionen innerhalb des Blocks Bescheid wissen und ein erneutes Senden dieser deshalb nicht nötig ist.
```

Comment:
```suggestion
Ein @Block besteht aus einem Header und @Transaktionen. Fluffy-Blocks enthalten lediglich einen Header, eine Liste von Transaktionsindizes und jegliche Transaktionen, welche der den Block empfangenden Node eventuell noch fehlen. Dies spart Bandbreite, weil Nodes möglicherweise bereits über die meisten Transaktionen innerhalb des Blocks Bescheid wissen und ein erneutes Senden dieser deshalb nicht nötig ist.
```

Reply:
Isn't "Indizes" basically the same as "Indexe"?
Your suggestion adds a dative case (would be "... welche dem den Block empfangenden Node eventuell noch fehlen."). I wasn't completely happy with my phrasing and with your suggestion it's still not satisfying. I worded it with a hypotaxis for easier/faster readability:
"Fluffy-Blocks enthalten lediglich einen Header, eine Liste von Transaktionsindexen und jegliche Transaktionen, welche dem Node, der den Block empfängt, eventuell fehlen. "
(Note that I can change "Indexe(n)" to Indizes" if preferred.)

Reply:
Plural of Index is almost impossible to get right.. could you take a look at the [Duden entry for Index](https://www.duden.de/rechtschreibung/Index)? It lists "Indizes" as plural of "Index" in the mathematical sense. You new, simplified translation looks good, except for the Indexen :)

Reply:
Thanks for the indeed very helpful link, I've learned something new today. :) 
Changed.